app/service_models/offer_validations.rb

```diff
@@ -0,0 +1,36 @@
+class OfferValidations
```

Comment:
Discussed in person but for posterity: we're going to move this to the services folder pending a wider discussion on where service models will live (ticket added to Dev team board https://trello.com/c/9jCgx5Lv/461-discuss-introducing-new-directory-to-seperate-validation-specific-to-services)